online_store/gt_sports.html

```diff
@@ -0,0 +1,100 @@
+ <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="gt_sports.css">
+    <title>goodTeens sports online store</title>
+  </head>
+  <body>
+<h2 class="column" id="h2title">goodTeens - Sports</h2>
+<h2 class="column" id="h2title2">Online Store</h2>
+<br><br><br><br><br>
+<p></p>
+<section class="card" id="product1"> 
+  <h3 class="productName1"></h3>
+</section>
+<section class="card" id="product2"> 
+  <h3 class="productName2"></h3>
+</section>
+<section class="card" id="product3"> 
+  <h3 class="productName3"></h3>
+</section>
+<section class="card" id="product4"> 
+  <h3 class="productName4"></h3>
+</section>
+
+<script>
+const sportProducts = [
+  {
+    imgSrc: "/skateboard.jpg",
+    name: "Skateboard",
+    brand: "Hudora",
+    level: "beginner",
+    price: 50,
+    seller: "Hudora",
+  },
+  {
+    imgSrc: "/standuppaddle.jpg",
+    name: "Stand-up paddle",
+    brand: "Oceana",
+    level: "advanced",
+    price: 300,
+    seller: "Oceana",
+  },
+  {
+    imgSrc: "/volleyball.jpg",
+    name: "Volleyball ball",
+    brand: "Mikasa",
+    level: "beginner",
+    price: 26,
+    seller: "Mikasa",
+    },
+  {
+    imgSrc: "/xbadminton.jpg",
+    name: "Badminton set",
+    brand: "Talbot Torro",
+    level: "intermediate",
+    price: 28,
+    seller: "Talbot Torro",
+    }
+];
+
+let product1 = document.getElementById("product1")
```

Comment:
again about indentation. Ex:
```javascript
<script>
    let product1 = document.getElementById("product1");
    product1 .innerHTML = `
        <p> <b>Name:</b> ${sportProducts[0].name}</p>
        <p> <img src='${sportProducts[0].imgSrc}'/><p>
        <p> <b>Brand:</b> ${sportProducts[0].brand}</p>
        <p> <b>Price:</b> ${sportProducts[0].price} Eur</p>
        <p> Sold by a <b>gT partner</b> called ${sportProducts[0].seller}.</p>
        <p> This ${sportProducts[0].name} is suitable for teenagers on ${sportProducts[0].level} level.
    `;
    ...
</script>
```

Reply:
Corrected. Get it. Need to get a better eye for that.